rpm-arm/Dockerfile

```diff
@@ -14,6 +18,7 @@ ENV DD_PIP_VERSION $DD_PIP_VERSION
 ENV DD_SETUPTOOLS_VERSION $DD_SETUPTOOLS_VERSION
 ENV CMAKE_VERSION $CMAKE_VERSION
 ENV CLANG_VERSION $CLANG_VERSION
+ENV GIMME_ARCH $TARGET_ARCH
```

Comment:
Isn't it supposed to be `$ARCH`? I don't see any `$GIMME_ARCH` used here, but I do see some `$ARCH` occurrences.